specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/examples/expressRouteProviderPort.json

```diff
@@ -0,0 +1,239 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "NetworkManagementClient",
+		"description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+		"version": "2021-08-01"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"security": [
+		{
+			"azure_auth": [
+				"user_impersonation"
+			]
+		}
+	],
+	"securityDefinitions": {
+    "azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow.",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteProviderPorts": {
```

Comment:
>"/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteProviderPorts" [](http://example.com/codeflow?start=4&length=87)

this api and the one below are identical, you cannot have two APIs with the same path. Just leave the one below with $filter as an optional parameter

Reply:
done